Python/Product/PythonTools/visualstudio_py_debugger.py

```diff
@@ -2318,7 +2318,7 @@ def detach_threads():
 
 def new_thread(tid = None, set_break = False, frame = None):
     # called during attach w/ a thread ID provided.
-    if tid == debugger_thread_id:
+    if tid and tid == debugger_thread_id:
```

Comment:
When testing now local debugging of Jython again, I detected the issue again. In fact, the expression ` tid == debugger_thread_id` evaluates in that case to `True` because also `debugger_thread_id` is `None`. This seems only to happen for local debugging. The previous change makes it running, but is not the right solution.